docs/services/slam/cartographer/_index.md

```diff
@@ -214,43 +214,30 @@ Select the **Raw JSON** mode, then copy/paste the following `"services"` and `"m
 
 ### Adjust `data_dir`
 
-Change the `data_dir` attribute to point to a directory on your machine where you want to save the sensor data your SLAM service uses and the maps and config files it produces.
+Change the `data_dir` attribute to point to a directory on your machine where you want to save the internal state your SLAM service produces.
 
 This directory must be structured as follows:
 
 <pre>
 .
 └──\(<file>CARTOGRAPHER_DIR</file>)
-    ├── <file>map</file>
-    ├── <file>data</file>
-    └── <file>config</file>
+    ├── <file>internal_state</file>
 </pre>
 
 Click through the following tabs to see the usage of each folder in this directory:
 
 {{% tabs name="Folders"%}}
-{{% tab name="/map" %}}
+{{% tab name="/internal_state" %}}
```

Comment:
Since we now only have one tab, please remove these two lines

Reply:
done